packages/react-redux-dynamic-reducer/.babelrc

```diff
@@ -1,3 +1,12 @@
 {
-  "presets": ["env", "react", "stage-3"]
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-syntax-import-meta",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-json-strings"
```

Comment:
what does `plugin-syntax-import-meta` and `plugin-proposal-json-strings` do?

Reply:
I used [babel-upgrade](https://github.com/babel/babel-upgrade) to do the upgrade, I think this replaces the stage-3 preset that is no longer supported.